src/ripple/test/jtx/quality.h

```diff
@@ -0,0 +1,52 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_TEST_JTX_QUALITY_H_INCLUDED
+#define RIPPLE_TEST_JTX_QUALITY_H_INCLUDED
+
+#include <ripple/test/jtx/Env.h>
+#include <ripple/test/jtx/tags.h>
+
+namespace ripple {
+namespace test {
+namespace jtx {
+
+/** Set the quality on a JTx. */
+class quality
```

Comment:
This quality class is not useful enough to put in src/ripple/test/jtx. In fact the same could be achieved with one or two calls to the JSON funclet:
```
..., json(sfQualityIn.fieldName, 1000000), json(sfQualityOut.fieldName, 40000)
```
Considering that almost no other tests would need it, I think it belongs in Path_test.cpp